includes/Plugin.php

```diff
@@ -162,6 +162,9 @@ function() use ( $options, $activation_flag ) {
 
 				$modules = new Core\Modules\Modules( $this->context, $options, $user_options, $authentication, $assets );
 				$modules->register();
+				if ( Feature_Flags::enabled( 'dashboardSharing' ) ) {
+					( new Core\Modules\REST_Dashboard_Sharing_Controller( $modules ) )->register();
+				}
```

Comment:
@jimmymadon, let's register this instance in the `Modules` class. See how we do it for the sharing settings instance: https://github.com/google/site-kit-wp/blob/00a2fdf067e3931df0ba20a20f7df3ff1f8789a1/includes/Core/Modules/Modules.php#L240

Reply:
@eugene-manuilov Of course, that makes more sense! I have created a new [PR here.](https://github.com/google/site-kit-wp/pull/5186) to not have the old commit.